src/store/gui/maintenance/actions.ts

```diff
@@ -92,7 +100,12 @@ export const actions: ActionTree<GuiMaintenanceState, RootState> = {
 
     async initStore({ commit, dispatch }, payload) {
         await commit('reset')
-        await commit('initStore', payload)
+
+        const entries = payload.value ?? {}
+        const initKey = Object.keys(entries).find((key) => entries[key]?.name === 'MAINTENANCE_INIT')
+        if (initKey) delete entries[initKey]
```

Comment:
`delete` sets the `MAINTENANCE_INIT`entry to `undefined`. is this intended?
to remove the entry the `splice()` function would be better

Reply:
entries is an object and not an array. i think splice only works on arrays.

Reply:
Ohh... you're right. Stupid.